docs-2.0/nebula-dashboard/deploy-dashboard.md

```diff
@@ -0,0 +1,195 @@
+# Deploy Dashboard
+
+The installation and deployment of Dashboard involve five kinds of services. This topic will describe how to install and deploy Dashboard in detail.
+
+## Port
+
+The deployment of Dashboard occupies the following ports:
+
+- 9200
+
+- 9100
+
+- 9090
+
+- 8090
+
+- 7003
+
+## Download Dashboard
+
+Download the configuration files for the deployment.
+
+```bash
+wget https://oss-cdn.nebula-graph.com.cn/nebula-graph-dashboard/{{dashboard.release}}.tar.gz
+```
+
+## Directory
+
+Run `tar -xvf {{dashboard.release}}.tar.gz` to decompress the installation package. There are 5 subdirectories in the `nebula-graph-dashboard`. The descriptions are as follows.
+
+|Name|Description|
+|:---|:---|
+|node-exporter | Collects the source information of machines in the cluster, including the CPU, memory, load, disk, and network.|
+|nebula-stats-exporter | Collects the performance metrics in the cluster, including the IP addresses, versions, and monitoring metrics (such as the number of queries, the latency of queries, the latency of heartbeats, and so on).|
+|prometheus | The time series database that stores monitoring data. |
+|nebula-http-gateway | Provides HTTP ports for cluster services to access the prometheus service or execute nGQL statements to interact with the Nebula Graph database.|
+|nebula-graph-dashboard| Provides the Dashboard service. Note that the name of this directory is the same as the root directory. The following `nebula-graph-dashboard` refers to the subdirectory.|
+
+The above five directories need to be duplicated to each machine according to deployment needs. The details are as follows.
+
+## Procedure
+
+### How to deploy the `node-exporter` service
```

Comment:
Why not use the verb + object form like `## Download Dashboard`? Consistency is important in technical writing.